src/man/firejail.txt

```diff
@@ -2858,7 +2871,6 @@ $ firejail \-\-tree
 #endif
 .br
   11970:netblue:transmission-gtk
-
```

Comment:
Suggestion: Undo this change as it's unrelated.

Other than that, LGTM.

Reply:
Done!